### PR TITLE
67 find the urbanrural classification data by output area for scotland

### LINKED ISSUE #249: Implement Urban/Rural Classification for Scotland.
The data/rural classification data with description can be found here #67 

Steps for Developer:
- Read data in as part of the pipeline
- Manipulate the data to create new urban/rural column based on rules in #67
- Merge onto dataset (some investigation needs to be done at which point) before the rural/urban aggregates are calculated.

### PR DESCRIPTION
**Pull request by Antonio-John**
This produces new estimates for transport accessibility disaggregated by urban/rural population.

The new data source is contained within data/urban_rural/scotland/
and can be downloaded [here](https://www.isdscotland.org/Products-and-Services/GPD-Support/Geography/Urban-Rural-Classification/).

I have uploaded these to the sync website.

I will wait for #287 to be closed before merging. There are likely to be conflicts, so I will solve them after that is merged then request a review from @paigeh1 afterwards.

Closes #249

### CODE REVIEW

**Comment by Antonio-John** [comment]
I have solved the conflicts & working. Ready for review whenever you have capacity. @paigeh1

File: SDG_scotland.py [hunk 1]
```diff
@@ -187,20 +231,129 @@
...
     # Output this iteration's sex df to the dict
     sex_df_dict[local_auth]=sex_servd_df_out
 
     # Output this iteration's df to the dict
     total_df_dict[local_auth] = la_results_df_out
+    
+
+    # Urban/Rural disaggregation
+    # split into two different dataframes
+    urb_df = only_la_pwc_with_pop[only_la_pwc_with_pop.urb_rur_class == "urban"]
```
Comment (paigeh1) [inline SDG_scotland.py:243]: A lot of this is a repeat of what it is in E&W. Can we make an urban rural function similar to disability?

**Review by paigeh1 — CHANGES_REQUESTED** [review]
Really great work on this! I have put a couple comments. As mentioned, I think there is duplicate code and it could be useful to have a function as mentioned to improve both scripts :)

**Review by paigeh1 — APPROVED** [review]
Happy to approve :) Can you put info in Nathan's document on the data you have found so it is in one place please? Don't think it is stored elsewhere

**Pull request merged**